cmd/config/init/utils.go

```diff
@@ -5,6 +5,6 @@ import (
 	"path/filepath"
 )
 
-func getRollerRootDir() string {
+func GetRollerRootDir() string {
```

Comment:
will this command only be used in the init package? or is it worth putting it in a higher-level package?